ckanext/datastore/db.py

```diff
@@ -102,7 +102,9 @@ def _get_engine(data_dict):
     engine = _engines.get(connection_url)
 
     if not engine:
-        engine = sqlalchemy.create_engine(connection_url)
+        import pylons
+        extras = {'url': connection_url}
+        engine = sqlalchemy.engine_from_config(pylons.config, 'ckan.datastore.sqlalchemy.', **extras)
```

Comment:
The tests run pep8 against the code, which complains about lines longer than 80 chars.  I know, I know, I'm not a fan either :(